packages/react-components/react-tabster/src/focus/createCustomFocusIndicatorStyle.ts

```diff
@@ -24,19 +25,9 @@ export interface CreateCustomFocusIndicatorStyleOptions {
  */
 export function createCustomFocusIndicatorStyle<TStyle extends GriffelStyle | GriffelResetStyle>(
   style: TStyle,
-  {
-    selector = defaultOptions.selector,
-    enableOutline = false,
-  }: CreateCustomFocusIndicatorStyleOptions = defaultOptions,
+  { selector = defaultOptions.selector }: CreateCustomFocusIndicatorStyleOptions = defaultOptions,
 ): TStyle extends GriffelStyle ? GriffelStyle : GriffelResetStyle {
   return {
-    ':focus': {
```

Comment:
maybe let's replace at least with `outlineStyle: 'transparent'` so that it will work by default in HC

Reply:
Ah, I actually remember why I didn't do this -- the only time `outlineStyle: none` is applied is when `getFocusOutlineStyles` is also applied with the border-based focus, which will also show up in high contrast mode OK.